packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -4,28 +4,70 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { Aliases, AuthInfo } from '@salesforce/core';
-import { ForceConfigGet } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
-
+import {
+  Aliases,
+  AuthInfo,
+  ConfigAggregator,
+  ConfigFile
+} from '@salesforce/core';
+import * as vscode from 'vscode';
 export class OrgAuthInfo {
   public static async getDefaultUsernameOrAlias(
     vscodePath: string
   ): Promise<string | undefined> {
-    const forceConfig = await new ForceConfigGet().getConfig(
+    if (
+      vscode.workspace &&
+      vscode.workspace.workspaceFolders &&
+      vscode.workspace.workspaceFolders[0]
+    ) {
+      const rootPath = vscode.workspace.workspaceFolders[0].uri.fsPath + '/';
+      const myLocalConfig = await ConfigFile.create({
+        isGlobal: false,
+        rootFolder: rootPath + '.sfdx/',
+        filename: 'sfdx-config.json'
+      });
+      const localDefault = myLocalConfig.get('defaultusername');
+      return JSON.stringify(localDefault).replace(/\"/g, '');
+    }
+
+    const aggregator = await ConfigAggregator.create();
+    const globalDefault = aggregator.getPropertyValue('defaultusername');
+    return JSON.stringify(globalDefault).replace('""', '');
```

Comment:
This should also be `.replace(/\"/g, '')`, I was getting double quotes when converting to a string so this is to remove those.